### PR TITLE
Add missing setting lms root url and release fixes for dogwood and eucalyptus images

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

The `LMS_ROOT_URL` setting is missing from dogwood and eucalyptus flavors settings.

This setting is required to compute absolute urls with protocol, domain and subdomains.

## Proposal

Add missing setting `LMS_ROOT_URL` in dogwood and eucalyptus flavors with a default value equal to the local address of the LMS as served in this project.

### CODE REVIEW

File: releases/dogwood/3/bare/CHANGELOG.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ release.
...
+## [dogwood.3-1.1.4] - 2019-12-19
+
 ### Fixed
 
+- Add missing setting `LMS_ROOT_URL` used to compute absolute urls
 - Fix broken CMS JS build by enabling Pipeline's static files storage
-
-### Added
-
 - Configure `general` cache backend including cache keys sanitizing function
```
Comment (sampaccoud) [inline releases/dogwood/3/bare/CHANGELOG.md]: :point_up: This change was actually a fix so moving it under "Fixed" and making a fix release.

File: releases/eucalyptus/3/bare/CHANGELOG.md [hunk 2]
```diff
@@ -9,16 +9,13 @@ release.
 
 ## [Unreleased]
 
-### Changed
-
-- Upgrade to nodejs 10 engine
-
-### Added
-
-- Configure `general` cache backend including cache keys sanitizing function
```
Comment (sampaccoud) [inline releases/eucalyptus/3/bare/CHANGELOG.md]: :point_up: All these changes were actually fixes so moving them under "Fixed" and making a fix release.

File: releases/master/bare/CHANGELOG.md [hunk 3]
```diff
@@ -17,4 +17,8 @@ release.
 
 - Set replicaSet and read_preference in mongodb connection
 
+### Fixed
+
+- Add missing setting `LMS_ROOT_URL` used to compute absolute urls
+
```
Comment (jmaupetit) [inline releases/master/bare/CHANGELOG.md]: In fact, this changelog is not maintained, and the previous added subsection should not have been committed (see the preamble of the file).

Comment (jmaupetit) [reply releases/master/bare/CHANGELOG.md]: I would remove the two subsections instead.

**Comment by rmoch** [comment]
I think this setting is only required by our flavor eucalyptus/wb

File: releases/hawthorn/1/bare/config/cms/docker_run_production.py [hunk 4]
```diff
@@ -124,7 +124,7 @@
 LMS_BASE = config("LMS_BASE", default="localhost:8072")
 CMS_BASE = config("CMS_BASE", default="localhost:8082")
 
-LMS_ROOT_URL = config("LMS_ROOT_URL", default="http://localhost:8072")
+LMS_ROOT_URL = config("LMS_ROOT_URL", default="http://{:s}".format(LMS_BASE))
```
Comment (jmaupetit) [inline releases/hawthorn/1/bare/config/cms/docker_run_production.py]: This has not been backported to the LMS settings

File: releases/hawthorn/1/oee/config/cms/docker_run_production.py [hunk 5]
```diff
@@ -125,7 +125,7 @@
 LMS_BASE = config("LMS_BASE", default="localhost:8072")
 CMS_BASE = config("CMS_BASE", default="localhost:8082")
 
-LMS_ROOT_URL = config("LMS_ROOT_URL", default="http://localhost:8072")
+LMS_ROOT_URL = config("LMS_ROOT_URL", default="http://{:s}".format(LMS_BASE))
```
Comment (jmaupetit) [inline releases/hawthorn/1/oee/config/cms/docker_run_production.py]: ditto

**Comment by jmaupetit** [comment]
> I think this setting is only required by our flavor eucalyptus/wb

@rmoch It was already set for hawthorn releases

**Comment by rmoch** [comment]
>> I think this setting is only required by our flavor eucalyptus/wb

> @rmoch It was already set for hawthorn releases

Ok, so it did not existed yet in dogwood. no reference in edx source code.

**Comment by sampaccoud** [comment]
@rmoch I searched it in the edx code base and found some occurrences.

**Comment by rmoch** [comment]
> @rmoch I searched it in the edx code base and found some occurrences.

ok then

**Comment by sampaccoud** [comment]
@rmoch @jmaupetit ready for final review!

**Pull request merged**